doc/developer/server_side.rst.mako

```diff
@@ -155,20 +91,15 @@ Object model
 
 ``FullTextSearch`` references a first level ``LayerGroup`` but without any constrains.
 
-it is not visible on this schema, but the ``User`` of a child schema has a link (``parent_role``)
-to the ``Role`` of the parent schema.
-
 ``metadata`` vs ``functionality``
 ....................................
 
 Technically the same ``functionality`` can be reused by more than one element.
 
 ``functionalities`` are designed to configure and customize various parts of
-the application. For instance to change the default basemap when a new theme
-is loaded.
+the application. For instance to change the default basemap when a new theme is loaded.
```

Comment:
Suggestion: "For instance, via a functionality you can define which basemap is to be used when a new theme is loaded."